analyzer/src/test/kotlin/managers/BundlerTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
+
+import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+
+import org.ossreviewtoolkit.model.Hash
+import org.ossreviewtoolkit.model.HashAlgorithm
+import org.ossreviewtoolkit.model.RemoteArtifact
+import org.ossreviewtoolkit.model.VcsInfo
+import org.ossreviewtoolkit.model.VcsType
+
+private val rpsecGemSpec = GemSpec(
+    name = "rspec",
+    version = "3.7.0",
+    homepageUrl = "http://github.com/rspec",
+    authors = sortedSetOf("Steven Baker", "David Chelimsky", "Myron Marston"),
+    declaredLicenses = sortedSetOf("MIT"),
+    description = "BDD for Ruby",
+    runtimeDependencies = setOf("rspec-core", "rspec-expectations", "rspec-mocks"),
+    vcs = VcsInfo(VcsType.GIT, "http://github.com/rspec/rspec.git", ""),
+    artifact = RemoteArtifact(
+        "https://rubygems.org/gems/rspec-3.7.0.gem",
+        Hash("0174cfbed780e42aa181227af623e2ae37511f20a2fdfec48b54f6cf4d7a6404", HashAlgorithm.SHA256)
+    )
+)
+
+class BundlerTest : WordSpec({
+    "creatFromJson()" should {
+        "correctly parse gem data retrieved from https://rubygems.org" {
```

Comment:
The test name is unclear about whether the test itself retrieves that data from https://rubygems.org, i.e. whether it makes an Internet connect. As a unit test, it should not, which is why I find the test name confusing.

Also, as this is the only test for now, maybe using a `StringSpec` instead of a `WordSpec` would be enough. The test name could then be something like

"createFromJson() parses JSON metadata for a Gem correctly"